util/container/Dockerfile

```diff
@@ -61,36 +61,24 @@ WORKDIR /tools
 COPY --from=openocd /tools/openocd openocd
 COPY --from=verilator /tools/verilator/${VERILATOR_VERSION} verilator/${VERILATOR_VERSION}
 
-# Required packages
-RUN apt-get update && apt-get install -y \
-    build-essential \
-    curl \
-    clang-format-3.8 \
-    git \
+# Required packages (from apt-requirements.txt)
+COPY apt-requirements.txt /tmp/apt-requirements.txt
+RUN apt-get update && \
+    sed 's/#.*//' /tmp/apt-requirements.txt \
+        | xargs apt-get install -y
+# Other packages (from before Dockerfile used apt-requirements.txt)
+RUN apt-get install -y \
     gnupg2 \
     libc6-i386 \
-    libelf-dev \
     libftdi-dev \
-    libftdi1-dev \
     libftdi1 \
-    libssl-dev \
     libtool \
     libusb-1.0-0-dev \
-    libxml2-dev \
     minicom \
-    ninja-build \
-    pkgconf \
-    screen \
-    srecord \
-    zlib1g-dev
-
-# Install Python 3 and support libraries. Cleanup install in place to reduce
-# binary size.
-RUN apt-get install -y \
-    python3 \
-    python3-pip \
-    python3-setuptools && \
-    apt-get clean; rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/share/doc/*
+    screen
+
+# Cleanup install in place to reduce binary size.
+RUN apt-get clean; rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/share/doc/*
```

Comment:
This needs to be part of the apt-get install part to avoid bloated intermediate layers.

Reply:
Done